docs/simple_l3_mirroring_wcm.md

```diff
@@ -0,0 +1,62 @@
+# simple_l3_mirroring_wcm
+
+## Overview
+This document describes how to build pb.bin file, load the pipeline and rules to program for the given P4 example on ES2000 target.
+Refer to ipdk-es2k.md as a reference document for 'P4C compilation', 'cpt packaging', component version details and other test execution details.
+
+## Lookup table 
+ExactMatchLookupTable is new P4 extern object type introduced to define profiles where key/keys and corresponding data field(s)
+are present. It differs from the existing Match-Action table in the sense that, it doesn't have any action associated with it. 
+Ideally this table can be configured with Exact or Ternary lookup keys. 
+This new table is defined as a user defined extern object type for the PNA architecture. The corresponding .proto changes that are 
+required to support this new table construct are introduced in p4runtime submodule, and the same needs to be open-sourced. 
+
+## SDE reference for p4 and README_OVS
+p4 program 
+https://github.com/intel-innersource/networking.ethernet.acceleration.vswitch.p4-sde.p4-driver/tree/main/mev_reference_p4_files/simple_l3_mirror_wcm
+
+README_OVS
```

Comment:
References to innersource

Reply:
I'm going to remove the document. It's for a P4 file that is not part of the networking recipe.